addons/l10n_mx/models/chart_template.py

```diff
@@ -0,0 +1,103 @@
+# coding: utf-8
+# Copyright 2016 Vauxoo (https://www.vauxoo.com) <info@vauxoo.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import models, api, _
+
+
+class AccountChartTemplate(models.Model):
+    _inherit = "account.chart.template"
+
+    @api.multi
+    def _load_template(
+            self, company, code_digits=None, transfer_account_id=None,
+            account_ref=None, taxes_ref=None):
+        self.ensure_one()
+        accounts, taxes = super(AccountChartTemplate, self)._load_template(
+            company, code_digits=code_digits,
+            transfer_account_id=transfer_account_id, account_ref=account_ref,
+            taxes_ref=taxes_ref)
+        account_tax_obj = self.env['account.tax']
```

Comment:
You are overwriting a method that will be called each time you install a tax chart if l10n_mx is installed. You can have an installation with l10n_mx and l10n_es installed and both shouldn't collide. I repeat: you should check that the account chart you're installing is really the Mexican one.